### DISCUSSION TITLE
Forms

### DISCUSSION THREAD

**Discussion by dvejsada** [twentyhq/twenty#8177] [Ideas]
Hi, do you plan to implement forms for better user experience when creating new table entries? Namely, we consider using Twenty for keeping client references, but for our employees (mostly attorneys with lower technical skills) it will be quite complicated to work with table view when adding new entries (especially as some entries may be longer texts). Having a nice form to give them via link will be significant step forward for easier adoption. Thanks!

**Comment by BOHEUS** [comment]
@dvejsada Twenty now offers forms in workflows, have you checked them out?

**Comment by BOHEUS** [comment]
Now when you create a new record, a form with all fields appears in side panel by default
